src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -748,15 +748,14 @@ static void window_scenery_periodic_update(rct_window* w)
 static void window_scenery_update(rct_window* w)
 {
     const CursorState* state = context_get_cursor_state();
-    rct_window* other = window_find_from_point(ScreenCoordsXY(state->x, state->y));
+    rct_window* other = window_find_from_point(state->position);
     if (other == w)
     {
-        int32_t window_x = state->x - w->x + 26;
-        int32_t window_y = state->y - w->y;
+        ScreenCoordsXY window = state->position - ScreenCoordsXY{ w->x + 26, w->y };
```

Comment:
This is subtly different. Should be `-26`

Reply:
Nice catch!